desktop/src/com/mygdx/game/client/input/TechnologyScreenUiInputAdapter.java

```diff
@@ -21,8 +21,8 @@ public TechnologyScreenUiInputAdapter(
 
   @Override
   public boolean keyDown(int keycode) {
-    switch (keycode) {
```

Comment:
zostaw jak było

Reply:
to mi analizator kod sam zmienia, nie chcę mi się za każdym razem walczyć z robotem jak będzie automatyczne czyszczenie kodu